eclipse-scout-core/src/keystroke/KeyStroke.js

```diff
@@ -138,9 +139,19 @@ export default class KeyStroke {
       if (this.field.visible !== undefined && !this.field.visible) {
         return false;
       }
-      // Check enabled state
-      if (this.field.enabled !== undefined && !this.field.enabled) {
-        return false;
+      // Check enabled state (if inheritAccessibility is true)
+      if (!this.inheritAccessibility) {
+        return true;
+      }
+      if (this.field.enabledComputed !== undefined) {
+        if (!this.field.enabledComputed) {
+          return false;
+        }
```

Comment:
just return this.field.enabledComputed here?

Reply:
much better, thanks